### ISSUE TITLE
Add electric vehicle charging station availability link to map marker

### ISSUE THREAD

**Issue by barbeau** [CUTR-at-USF/usf-mobullity#227]
More info on USF Electric Vehicle Charging stations:
http://www.usf.edu/administrative-services/parking/parking/ev-charging-stations.aspx

Apparently they use a company ChargePoint, which does provide real-time info on availability:
https://na.chargepoint.com/charge_point

At a minimum we should include a link to the above site in our map marker popups.  I'm going to see if I can track down a contact at the company to see if they have a developer API.

**Comment by barbeau** [comment]
Related to the Parking layer - https://github.com/CUTR-at-USF/usf-mobullity/issues/179.

**Comment by jmfield2** [comment]
The undocumented (as far as I can tell) API to see realtime info works like this:

```
https://na.chargepoint.com/dashboard/getChargeSpots?&lat=28.064100265365624&lng=-82.41240318864584&ne_lat=28.064277778199877&ne_lng=-82.41196800023317&sw_lat=28.063922752238177&sw_lng=-82.4128383770585&user_lat=0&user_lng=0&search_lat=28.0587031&search_lng=-82.41385389999999&scWidth=1346&scHeight=658&sort_by=distance&f_estimationfee=false&driver_filters=false&f_available=false&f_free=false&f_l1=false&f_l2=false&f_chademo=false&f_saecombo=false&f_tesla=false&f_cp=false&f_blink=false&f_semacharge=false&f_evgo=false&community=true&non_community=true&show_mode2_only=false&show_mode1_only=false&_=1458907548624
```

And returns JSON:

```
[{"station_list":{"network_info":{"13":{"name":"Greenlots","network_url":"http:\/\/greenlots.com\/","logo_url":"https:\/\/mc.chargepoint.com\/images\/network\/2\/ic_network_greenlots.png"}},"time":"2016-03-25 12:10:18.238","watermark":2313036183,"map_data_size":6,"summaries":[{"lat":28.064075469971,"lon":-82.41242980957,"port_count":{"total":2,"available":2},"station_status":"available","map_data":{"level1":{"free":{"available":1,"in_use":0,"total":1}},"level2":{"free":{"available":1,"in_use":0,"total":1}}},"device_id":3418,"payment_type":"free","tou_status":"open","can_remote_start_charge":true,"is_connected":false,"community_mode":0,"distance":994,"bearing":45,"port_type_count":{"1":1,"3":1},"station_name":["USF","EVSE 02"],"address":{"address1":"4202 E Fowler Ave","state_name":"Florida","city":"Tampa"},"description":"USF Cedar Cir. Parking Lot 5A, west side near lot entrance.","pricing":"pricing_free","station_power_shed_status":false,"tip_count":1,"reference_distance":2012,"reference_bearing":0,"distance_in_mile":"0.19","user_distance_in_mile":0,"reference_distance_in_mile":"0.38","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}},{"lat":28.064125061035,"lon":-82.412437438965,"port_count":{"total":2,"available":1},"station_status":"in_use","map_data":{"level1":{"free":{"available":1,"in_use":0,"total":1}},"level2":{"free":{"available":0,"in_use":1,"total":1}}},"device_id":7477,"payment_type":"free","tou_status":"open","can_remote_start_charge":true,"is_connected":false,"community_mode":0,"distance":1009,"bearing":45,"port_type_count":{"1":1,"3":1},"station_name":["USF","EVSE 01"],"address":{"address1":"4202 E Fowler Ave","state_name":"Florida","city":"Tampa"},"description":"USF Cedar Cir. Parking Lot 5A, west side near lot entrance.","pricing":"pricing_free","station_power_shed_status":false,"tip_count":1,"reference_distance":2029,"reference_bearing":0,"distance_in_mile":"0.19","user_distance_in_mile":0,"reference_distance_in_mile":"0.38","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}},{"lat":28.058963061314,"lon":-82.417354010967,"port_count":{"total":2,"available":1},"station_status":"available","map_data":{"level2":{"free":{"available":1,"in_use":1,"total":2}}},"device_id":117685,"payment_type":"free","tou_status":"open","can_remote_start_charge":true,"is_power_sharing":true,"is_connected":false,"community_mode":0,"distance":1494,"bearing":225,"port_type_count":{"3":2},"station_name":["USF","BEARD GARAGE 01"],"address":{"address1":"3800 USF Beard Drive","state_name":"Florida","city":"Tampa"},"description":"Beard Parking Garage, Level 1. Northwest Corner. Resident Permit Required.","pricing":"pricing_free","station_power_shed_status":false,"reference_distance":1130,"reference_bearing":270,"distance_in_mile":"0.28","user_distance_in_mile":0,"reference_distance_in_mile":"0.21","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}},{"lat":28.05894057526,"lon":-82.417356693176,"port_count":{"total":2,"available":1},"station_status":"available","map_data":{"level2":{"free":{"available":1,"in_use":1,"total":2}}},"device_id":119575,"payment_type":"free","tou_status":"open","can_remote_start_charge":true,"is_power_sharing":true,"is_connected":false,"community_mode":0,"distance":1501,"bearing":225,"port_type_count":{"3":2},"station_name":["USF","BEARD GARAGE 02"],"address":{"address1":"3800 USF Beard Drive","state_name":"Florida","city":"Tampa"},"description":"Beard Parking Garage, Level 1. Northwest Corner. Resident Permit Required.","pricing":"pricing_free","station_power_shed_status":false,"reference_distance":1130,"reference_bearing":270,"distance_in_mile":"0.28","user_distance_in_mile":0,"reference_distance_in_mile":"0.21","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}},{"lat":28.050153622031,"lon":-82.414488295753,"port_count":{"total":2,"available":2},"station_status":"available","map_data":{"level2":{"free":{"available":2,"in_use":0,"total":2}}},"device_id":120253,"payment_type":"free","tou_status":"open","can_remote_start_charge":true,"is_connected":false,"community_mode":0,"distance":4201,"bearing":180,"port_type_count":{"3":2},"station_name":["FL DOT 7","HEADQUARTERS 01"],"address":{"address1":"11201 N Malcolm McKinley Dr","state_name":"Florida","city":"Tampa"},"description":"South side of building adjacent carpool parking area","pricing":"pricing_free","station_power_shed_status":false,"tip_count":2,"reference_distance":3125,"reference_bearing":180,"distance_in_mile":"0.80","user_distance_in_mile":0,"reference_distance_in_mile":"0.59","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}},{"lat":28.050964355469,"lon":-82.39412689209,"port_count":{"total":2,"available":0},"station_status":"unknown_dc","map_data":{},"out_of_network":2,"device_id":1000021991,"tou_status":"open","distance":7452,"bearing":135,"port_type_count":{"83":2},"station_name":["City of Temple Terrace"],"address":{"address1":"11302 N 56th St","state_name":"FL","city":"Temple Terrace"},"description":"24 hours daily","network_id":13,"reference_distance":6949,"reference_bearing":135,"distance_in_mile":"1.41","user_distance_in_mile":0,"reference_distance_in_mile":"1.32","program_info":{"bmw_program_enabled":0,"bmw_program_enrolment_open":1,"nissan_program_enabled":0,"honda_program_enabled":0}}],"port_type_info":{"1":{"connector":"NEMA 5-20R","energy":1.9199999570847,"level":"Level 1","unit":"kW"},"3":{"connector":"J1772","energy":6.5999999046326,"level":"Level 2","unit":"kW"},"83":{"connector":"CHAdeMO","energy":0,"level":"DC","unit":"kW"}},"port_type_text_map":{"1":"NEMA 5-20R - 1.92kw","3":"J1772 - 6.6kw","83":"CHAdeMO"},"page_offset":"last_page","map_bound":{"ne_lat":28.065286551089,"ne_lon":-82.391803911981,"sw_lon":-82.419679673285,"sw_lat":28.048992131977}},"user_info":{"is_guest":1}}]
```

**Comment by jmfield2** [comment]
Fixed by ebcc1fe0805fe5cb981242138caeada4eeae5f7d
